exercises/concept/weather-forecast/.docs/instructions.md

```diff
@@ -1,21 +1,15 @@
 # Instructions
 
-Being hired by a big weather forecast company, your job is to maintain their code base. Scrolling through the code, you find it hard to follow what the code is actually doing. Good thing you know just the right thing to do!
+Goblinocus is a country that takes its weather forecast very seriously! Since you are a renowned responsible proficient developer, they have asked you to write a program for them that can forecast the current weather condition of various cities in Goblinocus. You were busy at that time and asked one of your friends to do the job instead. After a while, the president of Goblinocus contacted you and said they do not understand a bit about your friend's code! When you check the code, you find out your friend did not act as a responsible programmer and there are no comments in the code!!! Therefore as a responsible person, you feel obligated to clarify the program so goblins can understand them as well!
```

Comment:
```suggestion
Goblinocus is a country that takes its weather forecast very seriously! Since you are a renowned responsible and proficient developer, they have asked you to write a program for them that can forecast the current weather condition of various cities in Goblinocus. You were busy at that time and asked one of your friends to do the job instead. After a while, the president of Goblinocus contacted you and said they do not understand a bit about your friend's code! When you check the code, you find out your friend did not act as a responsible programmer and there are no comments in the code! Therefore as a responsible person, you feel obligated to clarify the program so goblins can understand them as well!
```

Suggested by Grammarly.